src/components/ReplyComment/index.js

```diff
@@ -92,7 +95,10 @@ const ReplyCommentId = ({
         </Text>
         <View style={styles.btn} />
       </View>
-      <ScrollView ref={scrollViewRef} contentContainerStyle={styles.commentScrollView}>
+      <ScrollView
+        onContentSizeChange={onSaveHeight}
+        ref={scrollViewRef}
+        contentContainerStyle={styles.commentScrollView}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onContentSizeChange` prop is added to the `ScrollView` component. This prop is a function that is called when the scrollable content view of the `ScrollView` changes. Make sure that the `onSaveHeight` function properly handles this event.
```diff
-        onContentSizeChange={onSaveHeight}
+        onContentSizeChange={onSaveHeight} // Ensure onSaveHeight function properly handles content size change
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->